mypy.ini

```diff
@@ -3,6 +3,7 @@ files = checkov
 exclude = checkov/(arm|cloudformation|common|dockerfile|github|bitbucket|gitlab|helm|json_doc|kubernetes|kustomize|sca_package|secrets|serverless|terraform)
 strict = True
 ignore_missing_imports = True
+show_error_codes = True
```

Comment:
nice! I checked it on google.
just out of curiosity, regarding the addition if the "error_code", do you have some concrete example in checkov where it help you to understand better the error?

Reply:
this is actually used for adding `# type: ignore` comments with a specific error code, so you only ignore a very specific error and not just turn type checking off for the whole line